### ISSUE TITLE
Decide on terminology for creators and donors/supporters

### ISSUE THREAD

**Issue by ErikBjare** [SuperuserLabs/superuserlabs.github.io#3]

**Comment by ahnlabb** [comment]
I strongly agree that supporter is better than donor. However, we need to make it clear what "supporter" entails for the doors to be effective, @powersource and I were discussing this yesterday and we couldn't really find a word that encapsulates it. What we want is "I want to be part of a crowdsourced patron".
